LINK/etc/profile.d/evm.sh

```diff
@@ -4,3 +4,16 @@
 alias ap='/usr/bin/appliance_console'
 alias vmdb='cd /var/www/miq/vmdb'
 alias appliance='[[ -n ${APPLIANCE_SOURCE_DIRECTORY} ]] && cd ${APPLIANCE_SOURCE_DIRECTORY}'
+
+# Tail MIQ Logs:
+function tailmiq()
+{
+  tail -f /var/www/miq/vmdb/log/$1.log
+}
+alias tailpglog='tail -f /var/opt/rh/rh-postgresql95/lib/pgsql/data/pg_log/postgresql.log'
```

Comment:
The PG_DATA directory changes whenever we change PG versions so we've created an environment variable so we only hardcode it in one place:  [APPLIANCE_PG_DATA](https://github.com/ManageIQ/manageiq-appliance/blob/73bac116a8805f24601c8fc1c7040487ecf91a67/LINK/etc/default/evm_postgres#L6)